File: PHP/1008.php
<?php

/* 
   * (🐺) Exercicio 1008 - Beecrowd - "Sálario"
*/

# Recebendo valores
$n_fuc = trim(fgets(STDIN));
$n_hrs = trim(fgets(STDIN));
$n_vhr = trim(fgets(STDIN));

# Cálculo do Sálario
$sal = $n_hrs * $n_vhr;

# Imprimindo argumento
printf("NUMBER = %d\n", $n_fuc);
printf("SALARY = U$ %.2f\n", $sal);

# ————————————————————————————————————————————————————————————————————————————— #
#                                                                               #
# --------------------- Código escrito por [Igor Rampazo] --------------------- #
# ------------------- GitHub:https://github.com/IgorRampazo ------------------- #
#                                                                               #
# ————————————————————————————————————————————————————————————————————————————— #

?>

File: C/1008.c
/* 
   * (🐺) Exercicio 1008 - Beecrowd - "Salário"
*/

// Inclusão de Arquivo(s) e Biblioteca(s)
#include <stdio.h>

int main()
{
   // Variáveis
   int n_fuc, n_hrs;
   float v_phr, sal;

   // Entrada de Dados
   scanf("%d", &n_fuc);
   scanf("%d", &n_hrs);
   scanf("%f", &v_phr);

   // Sálario
   sal = v_phr * n_hrs;

   // Imprimindo Resultado
   printf("NUMBER = %d\n", n_fuc);
   printf("SALARY = U$ %.2f\n", sal);
   
   return 0;
}

// ————————————————————————————————————————————————————————————————————————————— //
// ----------------------------------------------------------------------------- //
// --------------------- Código escrito por [Igor Rampazo] --------------------- //
// ------------------- GitHub:https://github.com/IgorRampazo ------------------- //
// ----------------------------------------------------------------------------- //
// ————————————————————————————————————————————————————————————————————————————— //